### ISSUE TITLE
free(): invalid size and socket hang up

### ISSUE THREAD

**Issue by agracia-foticos** [nuxt/nuxt#28718]
### Environment

- Operating System: `Linux`
- Node Version:     `v20.5.1`
- Nuxt Version:     `3.13.0`
- CLI Version:      `3.13.0`
- Nitro Version:    `2.9.7`
- Package Manager:  `yarn@4.3.1`
- Builder:          `-`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

When i launch npm run dev or yarn run dev raises
![image](https://github.com/user-attachments/assets/105e4c1c-364d-4ff5-a3c8-96767d6e4162)

### Describe the bug

When i launch npm run dev or yarn run dev raises
![image](https://github.com/user-attachments/assets/105e4c1c-364d-4ff5-a3c8-96767d6e4162)

**Comment by agracia-foticos** [comment]
with bun run dev also fails

**Comment by agracia-foticos** [comment]
i upgrade node to 22.7.0 and the error changes


![image](https://github.com/user-attachments/assets/03cb4479-f823-45e9-9880-6bee96162f41)

**Comment by agracia-foticos** [comment]
in another machine with node v20.5.1
![image](https://github.com/user-attachments/assets/df8fd20a-1bca-42ef-8126-7003cf640455)

**Comment by danielroe** [comment]
seems it wasn't a nuxt issue: https://github.com/nodejs/node/issues/54590#issuecomment-2312241251

**Comment by rustavellyne** [comment]
got the same isssue on Linux Fedora node 20 or 22, @agracia-foticos have you resolved it on your enviroment?

**Comment by agracia-foticos** [comment]
> got the same isssue on Linux Fedora node 20 or 22, [@agracia-foticos](https://github.com/agracia-foticos) have you resolved it on your enviroment?

No, i have mono repo with different projects and sometimes raises the error

**Comment by rustavellyne** [comment]
@victortolbert after disabling almost all components found that somehow 
`import config from '~/assets/static/config-footer.json'` cause that error

**Comment by nsch0e** [comment]
perhaps you can try to build dependencies from source. That fixed it for me.
```bash
npm install --build-from-source
```

**Comment by agracia-foticos** [comment]
Its random, not always raises the error... maybe its a problem with local memory

**Comment by rustavellyne** [comment]
I found in my case the reason was this module `import vsharp from 'vite-plugin-vsharp'`, when i disabled it, everything works fine
